static/css/com/layout/layout.scss

```diff
@@ -11,11 +11,13 @@
 
 .global-layout {
     min-height: 100%;
-    margin-bottom: -$footer-height;
```

Comment:
Oh, i know thats ugly, try to open [security.html](https://kotlinlang.org/security.html) or other short page.
I think it's better to improve body size :(

Reply:
Fixed it. Flex box forever ❤️